tests/test_dataset_versions_endpoint.py

```diff
@@ -52,11 +56,13 @@ def should_return_required_property_error_when_missing_mandatory_id_property() -
 def should_return_error_if_dataset_id_does_not_exist_in_db() -> None:
     body = {"id": any_dataset_id(), "metadata-url": any_s3_url()}
```

Comment:
`metadata-url` should be `metadata_url`.

Reply:
Should be fixed in 79753324dbc409a9d501a77203fc820c27be2c46